webpack/scenes/AlternateContentSources/Create/Steps/ACSSmartProxies.js

```diff
@@ -35,7 +42,17 @@ const ACSSmartProxies = () => {
         removeSelected={onListChange}
         id="selector"
       />
-      <FormGroup label={__('Use HTTP proxies')} fieldId="use_http_proxies">
+      <FormGroup
+        label={
+          <Flex spaceItems={{ default: 'spaceItemsNone' }}>
+            <FlexItem>{__('Use HTTP proxies')}</FlexItem>
+            <FlexItem>
+              {HelpToolTip(__('Alternate content sources use the HTTP proxy of their assigned smart proxy for communication.'))}
```

Comment:
This should be rendered normally - `<HelpToolTip ...` and not `{HelpToolTip(...`